server/backend/api/app/crud/crud_user_role.py

```diff
@@ -117,12 +117,18 @@ def find_or_create(db: Session, user_id: int, role_id: int):
         db.flush()
         LOGGER.debug(f"New FamUserRoleXref added for {new_fam_user_role.__dict__}")
         return new_fam_user_role
-
-    LOGGER.debug(
-        "FamUserRoleXref already exists with id: " +
-        f"{fam_user_role_xref.user_role_xref_id}."
-    )
-    return fam_user_role_xref
+    else:
+        LOGGER.debug(
+            "FamUserRoleXref already exists with id: " +
+            f"{fam_user_role_xref.user_role_xref_id}."
+        )
+        error_msg = f'{"Role already assigned to user."}'
+        crud_utils.raise_http_exception(HTTPStatus.CONFLICT, error_msg)
+    # LOGGER.debug(
+    #     "FamUserRoleXref already exists with id: " +
+    #     f"{fam_user_role_xref.user_role_xref_id}."
+    # )
+    # return fam_user_role_xref
```

Comment:
Hi Nick,
Should this commented out code being removed?

Reply:
Thanks. I am going to remove that.